src/pyhf/constraints.py

```diff
@@ -75,10 +77,16 @@ def _precompute(self):
         if not self.param_viewer.index_selection:
             return
         tensorlib, _ = get_backend()
-        self.batched_sigmas = tensorlib.astensor(self._batched_sigmas)
+        self.sigmas = tensorlib.astensor(self._sigmas)
         self.normal_data = tensorlib.astensor(self._normal_data, dtype='int')
         self.access_field = tensorlib.astensor(self._access_field, dtype='int')
 
+    def _dataprojection(self, auxdata):
```

Comment:
we should have a comment that this returns only the auxiliary data associated with the modifier -- and you should be calling the signature `def _dataproject(self, data)` since you normally pass in all data, and extract the auxdata from it

Reply:
`_dataprojection` will be removed in subsequent PRs